packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -4,6 +4,34 @@
     // Runner Configuration
     // ====================
     //<%
+    if(answers.generateTSConfigFile && hasFile('test/tsconfig.json')){ %> 
```

Comment:
Are we not supposed to generate this part no matter where the `tsconfig.json` is ?

For instance, if it's a new project, the `tsconfig.json` will be at the root of the project but it would still require `autoCompileOpts`, don't you think ?

Reply:
It picks up that automatically if it's in root folder , only if it's in non root location then only we need to specify the tsconfig location